tweaser/version.go

```diff
@@ -0,0 +1,9 @@
+package tweaser
+
+// Version is the main version number that is being run at the moment.
+const Version = "No version set"
```

Comment:
Why do we even need this package? The versions are being set directly by the ldflags, e.g. `actions.version`. Is it just for defaults?

Reply:
yeah - but im changing this up, wait for it, wait for it, wait fo........ 💤